vars/getFunctionalTags.groovy

```diff
@@ -0,0 +1,59 @@
+// vars/getFunctionalTags.groovy
+
+/**
+ * getFunctionalTags.groovy
+ *
+ * Get the avocado test tags for the functional test stage.
+ *
+ * @param kwargs Map containing the following optional arguments:
+ *      pragma_suffix   commit pragma suffix for this stage, e.g. '-hw-large'
+ *      stage_tags      launch.py tags to use to limit functional tests to those that fit the stage
+ *      default_tags    launch.py tag argument to use when no parameter or commit pragma tags exist
+ * @return String of test tags to run in the stage
+ */
+Map call(Map kwargs = [:]) {
```

Comment:
I wonder if I should add optional arguments taht would allow more control over which tags would be used under each started by case: user, upstream, and timer.